web-app/components/pages/login/login.jsx

```diff
@@ -24,117 +23,97 @@ const StyledGrid = styled(Grid)(() => ({
     justifyContent: 'space-around',
 }));
 
+const BoxImage = styled(Box)(() => ({
+    display: 'grid',
+    placeItems: 'center',
+}));
+
+const StyledImage = {
+    backgroundColor: 'rgb(234, 245, 247)',
+};
+
+const BoxLogin = styled(Box)(() => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '15px',
+    textAlign: 'center',
+    position: 'relative',
+    padding: '30px 0px 0px 0px',
+}));
+
+// For Input text
+const InputField = (props) => <StyledTextField variant="outlined" required size="small" {...props} />;
```

Comment:
Put this after the definition of `StyledTextField`.  Javascript can read the bottom up like this, but top down definition is way better approach